calculator/calculator.js

```diff
@@ -1,104 +1,201 @@
 import { question } from 'readline-sync';
 
-function getInput(call, typeOfOperation) {
+function sleep(milliseconds) {
+  return new Promise(resolve => setTimeout(resolve, milliseconds));
+}
+
+function getInput(call, operationType) {
   let ordinalNumber;
-  if (typeOfOperation != 'squareRoot') {
+
+  if (operationType != 'squareRoot') {
     if (call == 1) {
-      ordinalNumber = "primer";
+      ordinalNumber = 'primer';
     }
     else if (call == 2) {
-      ordinalNumber = "segundo";
+      ordinalNumber = 'segundo';
     }
     const input = question('Ingresa el ' + ordinalNumber + ' número: ');
     return input;
   }
-  else if (typeOfOperation == 'squareRoot') {
+  else if (operationType == 'squareRoot') {
     const input = question('Ingresa el número: ');
     return input;
   }
 }
 
+function showOutput(operationType, output) {
+  let message;
+
+  switch (operationType) {
+    case 'addition':
+      message = 'El resultado de la suma es: ' + output;
+      break;
+    case 'substraction':
+      message = 'El resultado de la resta es: ' + output;
+      break;
+    case 'multiplication':
+      message = 'El resultado de la multiplicación es: ' + output;
+      break;
+    case 'division':
+      message = 'El resultado de la división es: ' + output;
+      break;
+    case 'squareRoot':
+      message = 'Su raíz cuadrada es: ' + output;
+      break;
+    case 'power':
+      message = 'El resultado de la suma es: ' + output;
+      break;
+    case 'remainder':
+      message = 'El residuo es: ' + output;
+      break;
+    case 'compare':
+      message = 'El número mayor es: ' + output;
+      break;
+    default:
+      message = 'resultado inesperado';
+  }
+  return message;
+}
+
 function add(firstValue, secondValue) {
   let addition = parseInt(firstValue) + parseInt(secondValue)
+
   return addition;
 }
 
 function sustract(firstValue, secondValue) {
   let sustraction = parseInt(firstValue) - parseInt(secondValue);
+
   return sustraction;
 }
 
 function multiply(firstValue, secondValue) {
   let multiplication = parseInt(firstValue) * parseInt(secondValue);
+
   return multiplication;
 }
 
 function divide(firstValue, secondValue) {
   let division = parseInt(firstValue) / parseInt(secondValue);
+
   return division;
 }
 
 function getSquareRoot(value) {
   let squareRoot = parseFloat(Math.sqrt(value));
+
   return squareRoot;
 }
 
 function getPower(value, powerValue) {
   let power = parseInt(value) ** parseInt(powerValue);
+
   return power;
 }
 
-function calculate() {
+function getRemainder(firstValue, secondValue) {
+  let remainder = firstValue % secondValue;
+
+  return remainder;
+}

```

Comment:
```suggestion
function getRemainder(firstValue, secondValue) {
  return firstValue % secondValue;;
}

```

^^si no haces nd con la variable despues de asignarla, no es necesaria, pon el calculo directamente en el return

Reply:
con el objetivo de yo entender como funciona, Emma me solicitó hacerlo de esta amnera, más descriptiva y yo saber que es lo que está regresando la función.